__tests__/app.test.js

```diff
@@ -7,143 +7,215 @@ const app = require("../app");
 afterAll(() => db.end());
 beforeEach(() => seed(testData));
 
-describe("GET /api/topics", () => {
-  test("200: returns an array", async () => {
-    const res = await request(app).get("/api/topics").expect(200);
-    expect(res.body.length).toBe(3);
-    res.body.forEach((topic) => {
-      expect(topic).toEqual({
-        slug: expect.any(String),
-        description: expect.any(String),
+describe("TOPICS", () => {
+  describe("GET /api/topics", () => {
+    test("200: returns an array", async () => {
+      const res = await request(app).get("/api/topics").expect(200);
+      expect(res.body.length).toBe(3);
+      res.body.forEach((topic) => {
+        expect(topic).toMatchObject({
+          slug: expect.any(String),
+          description: expect.any(String),
+        });
       });
     });
-  });
-  test("404: incorrect path returns 'path not found'", async () => {
-    await request(app).get("/api/topic").expect(404);
+    test("404: incorrect path returns 'path not found'", async () => {
+      await request(app).get("/api/topic").expect(404);
```

Comment:
You're expecting the 404, which is great. But your test description expects a message of 'path not found' - how can you test for this?

Reply:
Got it, thanks and updated

Reply:
I think what Joey was getting at is that we should also test that the message we receive back matches "path not found" specifically. Consider adding a .then to your request chain to check what you receive against what you expect!